test/shared/base/view.test.js

```diff
@@ -325,13 +326,14 @@ describe('BaseView', function() {
         });
 
         it('it should create an instance of the collection', function () {
-          var result = BaseView.parseModelAndCollection(modelUtils, { collection: [], collection_name: 'MyCollection', app: this.app, collection_params: params });
+          var result = BaseView.parseModelAndCollection(modelUtils, { collection: [], collection_name: 'MyCollection', app: this.app, collection_params: params, parse: true });
 
           result.should.deep.equal({
             collection_name: 'MyCollection',
             collection_params: params,
             collection: collection,
-            app: this.app
+            app: this.app,
+            parse: true
```

Comment:
we should probably have a test that makes sure that it doesn't call `parse` as well.